doc/events.md

```diff
@@ -0,0 +1,150 @@
+# Events
+
+## Introduction
+Events are messages about actions that have occurred over LXD. Using the API endpoint `/1.0/events` directly or via
+`lxc monitor` will connect to a WebSocket through which logs and lifecycle messages will be streamed.
+
+## Event types
+LXD Currently supports three event types.
+
+- **Logging**: Shows all logging messages regardless of the server logging level.
+- **Operation**: Shows all ongoing operations from creation to completion (including updates to their state and progress metadata).
+- **Lifecycle**: Shows an audit trail for specific actions occurring over LXD.
+
+## Event structure
+#### Example:
+```yaml
+location: cluster_name
+metadata:
+  action: network-updated
+  requestor:
+    protocol: unix
+    username: root
+  source: /1.0/networks/lxdbr0
+timestamp: "2021-03-14T00:00:00Z"
+type: lifecycle
+```
+- `location`: The cluster member name (if clustered).
+- `timestamp`: Time that the event occurred in RFC3339 format.
+- `type`: The type of event this is (one of `logging`, `operation`, or `lifecycle`).
+- `metadata`: Information about the specific event type.
+
+#### Logging event structure
+- `message`: The log message.
+- `level`: The log-level of the log.
+- `context`: Additional information included in the event.
+
+#### Operation event structure
+- `id`: The UUID of the operation.
+- `class`: The type of operation (task, token, or websocket).
+- `description`: A description of the operation.
+- `created_at`: The operation's creation date.
+- `updated_at`: The operation's date of last change.
+- `status`: The current state of the operation.
+- `status_code`: The operation status code.
+- `resources`: Resources affected by this operation.
+- `metadata`: Operation specific metadata.
+- `may_cancel`: Whether the operation may be cancelled.
+- `err`: Error message of the operation.
+- `location`: The cluster member name (if clustered).
+
+#### Lifecycle event structure
+- `action`: The lifecycle action that occurred.
+- `requestor`: Information about who is making the request (if applicable).
+- `source`: Path to what is being acted upon.
+- `context`: Additional information included in the event.
+
+## Supported lifecycle events
+| Name                              | Description                                                           | Additional Information                                                                               |
+| :-------------------------------- | :-------------------------------------------------------------------- | :--------------------------------------------------------------------------------------------------- |
+| CertificateCreated                | A new certificate has been added to the server trust store.           |                                                                                                      |
```

Comment:
@masnax 

Oh, we need the event name, not the internal type, so `certificate-created` instead of `CertificateCreated`.
Same for all of the other ones.

Reply:
Oops, silly me :)